horovod/tensorflow/mpi_ops.py

```diff
@@ -168,3 +170,54 @@ def _broadcast_grad(op, grad):
     if rank() != root_rank:
         return grad_reduced * 0
     return grad_reduced
+
+
+def size_op(name=None):
```

Comment:
Can you add comments explaining when you would use this op instead of simply calling hvd.size(), etc.?